packages/editor/templates/Crypto Pizzas/README.md

```diff
@@ -38,7 +38,7 @@ Second it uses a [function modifier](https://solidity.readthedocs.io/en/v0.5.12/
 
 Once the modifier confirms the Pizza is unique the function adds it to the array and maps it to the owner (creator in this case), using [the `assert` error handling function](https://solidity.readthedocs.io/en/v0.5.12/control-structures.html#id4) to check that the owner address is the same as the address of the current user.
 
-The `createRandomPizza` function is the public function called in JavaScript that assigns the string the user sets in the front end as the name of the pizza and calls the `generateRandomDna` function, passing the name, and the owner.
+The `createRandomPizza` function is the public function called in JavaScript that assigns the string the user sets in the frontend as the name of the pizza and calls the `generateRandomDna` function, passing the name, and the owner.
```

Comment:
@samajammin 

https://www.sitepoint.com/spelling-frontend-front-end-front-etc/

😆

I have no idea, and don't really care, as long as we're consistent.